.github/workflows/deploy-book.yml

```diff
@@ -24,6 +24,12 @@ env:
   # By default, preprocess no branches ('').
   # Preprocessing no branches can also be set by ' '.
   BRANCHES_TO_PREPROCESS: ${{ vars.BRANCHES_TO_PREPROCESS }}
+  # If to copy the primary page to root
+  COPY: ${{ vars.COPY != '' && vars.COPY || 'false' }}
+  # Space-separated list of branch names
+  # Note: does not support '*'
```

Comment:
add default value

Reply:
The default value as of now is `false`, so redirects-only

Reply:
Now added default value

Reply:
I think we decided that main should be the default value, as it doesn't do the crazy symlinking but gives output at root